mchf-eclipse/drivers/audio/audio_driver.h

```diff
@@ -184,6 +185,8 @@ typedef struct AudioDriverState
     float32_t               onem_mtauR;
     float32_t               mtauI; //(exp(- DF / (IQ_SAMPLE_RATE_F * tauI))); //0.99999255955;
     float32_t               onem_mtauI;
+    float32_t               c0[7];          // Filter coefficients - path 0
```

Comment:
please replace 7 with STAGES 
probably STAGES needs to be defined in the header then